velox/dwio/common/BitPackDecoderUtil.h

```diff
@@ -0,0 +1,600 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/vector/TypeAliases.h"
+
+#include <folly/Range.h>
+#include <xsimd/config/xsimd_config.hpp>
```

Comment:
Keep the `@manual` as it is needed in our internal build system

Reply:
fixed